src/query/api/v1/handler/prometheus/handleroptions/headers.go

```diff
@@ -49,6 +49,19 @@ const (
 	// Valid values are "unaggregated" or "aggregated".
 	MetricsTypeHeader = M3HeaderPrefix + "Metrics-Type"
 
+	// WriteTypeHeader sets whether to perform a default write that can be
+	// written to unaggreated namespace and optionally aggregated or
+	// to specifically only perform aggregations (and skip writing unaggregated
+	// values).
+	// Valid values are "default" or "aggregate"
+	WriteTypeHeader = "M3-Write-Type"
+
+	// DefaultWriteType is the default write type.
```

Comment:
nit: maybe add the description here, i.e. `DefaultWriteType is the default write type. This writes to both aggregated and unaggregated namespaces.`